### ISSUE TITLE
GCUIGLOBALS.GLOBAL.MBIN - Unknown Exception

### ISSUE THREAD

**Issue by D-A-R-K-N-E-S-S** [monkeyman192/MBINCompiler#58]
I just updated to the latest experimental [8/15 Build# 3034427] and MBINCompiler [1.55.0-pre3].

When decompiling GCUIGLOBALS.GLOBAL.MBIN an unknown exception is thrown. I decompiled several other global files without any issue. Seems to be related to this particular file.

```
ERROR: An unknown exception occurred!
Index was outside the bounds of the array.
Stacktrace:
   at libMBIN.Models.NMSTemplate.SerializeEXmlValue(Type fieldType, FieldInfo fi
eld, NMSAttribute settings, Object value)
   at libMBIN.Models.NMSTemplate.SerializeEXml(Boolean isChildTemplate)
   at libMBIN.Models.NMSTemplate.SerializeEXmlValue(Type fieldType, FieldInfo fi
eld, NMSAttribute settings, Object value)
   at libMBIN.Models.NMSTemplate.SerializeEXml(Boolean isChildTemplate)
   at libMBIN.EXmlFile.WriteTemplate(NMSTemplate template)
   at MBINCompiler.Program.DecompileFile(String inputPath, String outputPath, Bo
olean getVersion, Boolean verbose)
   at MBINCompiler.Program.Main(String[] args)

Press any key
```

**Comment by monkeyman192** [comment]
Yep, I know this file has changed. I will get it decompiling tonight when I get time (same with other broken globals...)

**Comment by monkeyman192** [comment]
Fixed in 26cbacfdafce20e00e06632441595b7276dcf8df
